playbooks/robusta_playbooks/helm_release_events.py

```diff
@@ -0,0 +1,55 @@
+from typing import Optional
+
+from robusta.api import (
+    ActionParams,
+    action,
+    HelmReleasesChangeEvent,
+    Finding, FindingSource
+)
+import logging
+
+from robusta.core.reporting import MarkdownBlock
+
+
+class CreateHelmStatusNotificationParams(ActionParams):
+    message: Optional[str] = ""
+
+
+@action
+def create_helm_status_notification(event: HelmReleasesChangeEvent, params: CreateHelmStatusNotificationParams):
+    for helm_release in event.helm_releases:
+        logging.info(f"received - helm releases change event: {helm_release.namespace}/{helm_release.name}")
+
+        title = f"{params.message if params.message else f'Helm release - status: {helm_release.info.status}'}"
```

Comment:
I think the status should have high level information of this issue:
Something like:
Helm release {{ release name }} version {{ release version }} **failed**

Reply:
done